python/ray/serve/_private/constants.py

```diff
@@ -131,7 +131,7 @@
 )
 
 # Timeout for GCS RPC request
-RAY_GCS_RPC_TIMEOUT_S = 3.0
+RAY_GCS_RPC_TIMEOUT_S = 30.0
```

Comment:
get_all_node_ids is in our main event loop, it is a sync call. We probably need to consider to make it async in this case.